packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -406,24 +406,22 @@ export function makeVirtualObjectManager(
         harden(target);
       }
 
-      let representative;
+      let instanceKit;
       if (initializing) {
         innerSelf.wrapData = wrapData;
-        representative = instanceMaker(innerSelf.rawData);
+        instanceKit = harden(instanceMaker(innerSelf.rawData));
       } else {
         const activeData = {};
         wrapData(activeData);
-        representative = instanceMaker(activeData);
-        delete representative.initialize;
-        harden(representative);
+        instanceKit = harden(instanceMaker(activeData));
       }
       cache.remember(innerSelf);
-      valToSlotTable.set(representative, innerSelf.vobjID);
-      return representative;
+      valToSlotTable.set(instanceKit.self, innerSelf.vobjID);
+      return instanceKit;
     }
 
     function reanimate(vobjID) {
-      return makeRepresentative(cache.lookup(vobjID), false);
+      return makeRepresentative(cache.lookup(vobjID), false).self;
```

Comment:
Since it now makes an instance kit, how about?
```suggestion
      return makeInstanceKit(cache.lookup(vobjID), false).self;
```